targets/frontend/src/pages/glossary/edit/[[...id]].js

```diff
@@ -0,0 +1,278 @@
+                  />
+                  Supprimer le terme
+                </Button>
+              </div>
+            </>
+          )}
+          <TermForm term={term} />
+        </>
+      )}
+    </Layout>
+  );
+}
+
+export default withCustomUrqlClient(withUserProvider(EditTermPage));
+
+const TermForm = ({ term = {} }) => {
+  const { isAdmin } = useUser();
+  const router = useRouter();
+  const { control, register, handleSubmit, errors } = useForm();
+  const [editResult, editTerm] = useMutation(editTermMutation);
+  const [duplicateTermError, setDuplicateTermError] = useState(false);
+
+  async function onSubmit(data) {
+    const result = await editTerm({
+      term: {
+        ...(term.id && { id: term.id }),
+        ...data,
+      },
+    });
+    if (!result.error) {
+      router.push("/glossary");
+    } else if (
+      result.error.graphQLErrors[0].extensions.code === "constraint-violation"
+    ) {
+      setDuplicateTermError(true);
+    }
+  }
+
+  const loading = editResult.fetching;
+  const hasError = Object.keys(errors).length > 0;
+  let buttonLabel = "Créer";
+  if (term.id) {
+    buttonLabel = "Enregistrer les changements";
+  }
+  return (
+    <form onSubmit={handleSubmit(onSubmit)}>
+      <>
+        <div sx={{ mb: "small" }}>
+          <Field
+            disabled={!isAdmin}
+            type="text"
+            name="term"
+            label="Terme"
+            onChange={() => setDuplicateTermError(false)}
+            defaultValue={term.term}
+            ref={register({
+              required: { message: "Ce champ est requis", value: true },
+            })}
+          />
+          <FormErrorMessage errors={errors} fieldName="term" />
+          {duplicateTermError && (
+            <div sx={{ color: "critical" }}>Ce terme existe déjà !</div>
+          )}
+        </div>
+
+        <div sx={{ mb: "small" }}>
+          <Label htmlFor={"definition"}>Définition</Label>
+          <Textarea
+            disabled={!isAdmin}
+            name="definition"
+            id="definition"
+            rows={5}
+            defaultValue={term.definition}
+            ref={register({
+              required: { message: "Ce champ est requis", value: true },
+            })}
+          />
+        </div>
+
+        <Flex
+          sx={{
+            alignItems: "flex-start",
+            flexWrap: "wrap",
+            gap: "small",
+            justifyContent: "stretch",
+            mb: "small",
+          }}
+        >
+          <div sx={{ flex: "1 1 auto" }}>
+            <Label htmlFor={"variants"}>Variantes / Synonymes</Label>
+            <Lister
+              control={control}
+              disabled={!isAdmin}
+              name="variants"
+              id="variants"
+              defaultValue={term.variants}
+            />
+          </div>
+          <div sx={{ flex: "1 1 auto" }}>
+            <Label htmlFor={"abbreviations"}>Abréviations</Label>
+            <Lister
```

Comment:
Sortable component is not usefull (it has no impact glossary replacement)

Reply:
Is there any reason you would like to remove it ? (except to have less code ?)

Reply:
don't see any reason why it should stay ;)

Reply:
To have a list you can reorder ? It might not be super useful here but since it's reusable component I find it weird not to be able to reorder items inside